webapp/packages/core-sdk/src/queries/fragments/DatabaseConnection.gql

```diff
@@ -6,6 +6,8 @@ fragment DatabaseConnection on ConnectionInfo {
   driverId
   keepAliveInterval
   autocommit
+  defaultCatalogName
+  defaultSchemaName
```

Comment:
We don't want to load this data for every connection, we need this data only in connection form (you can re-use customIncludeOptions)

Reply:
Its the same as keepAliveInterval and autocommit

Reply:
We dont want to rewrite how we load expert settings here

Reply:
Especially when we are going to rewrite it in dynamic properties in the next sperint

Reply:
You're right; we don't need to load `keepAliveInterval` and `autocommit`, so let's change them, too.
We don't need to "rewrite" anything; we just need to pass the right flags in the right places, with everything already implemented.

We don't want to load information that we don't need.